### PR TITLE
Footer Layout

### PR DESCRIPTION
**Pull request by UX-Chris**
Description
-----------

Remove the about us from the footer, add a Contact Us button in the footer and add social media icons.

Motivation and Context
----------------------

Visual improvements
https://github.com/request-yo-racks/web/issues/62

How Has This Been Tested?
-------------------------

Screenshots:
------------

image

Types of changes
----------------

- New feature (non-breaking change which adds functionality)

Checklist:
----------

-  [] I have updated the documentation accordingly.
-  [] I have written unit tests

Fixes request-yo-racks/web#62

### CODE REVIEW

**Comment by rgreinho** [comment]
Footer view on a Galaxy S5 or Pixel phone (from the chrome developer tools):
image

File: src/my-app.html [hunk 1]
```diff
@@ -183,15 +186,15 @@
         <footer>
           <!-- <div class="app-grid"> -->
             <div class="item footer-left">
-              <h3>Contact us</h3>
+              <h3>Contact Us</h3>
```
Comment (rgreinho) [inline src/my-app.html]: Question: Why mot using a [paper button](https://www.webcomponents.org/element/PolymerElements/paper-button) element here?

**Comment by UX-Chris** [comment]
That all makes sense. I won't make it today but I'll work on those changes tomorrow.

**Comment by rgreinho** [comment]
We messed up the rebasing. Closing this one. Please refer to #69 for the correct version.

**Pull request closed without merging**